<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hakkinda</class>
 <widget class="QMainWindow" name="hakkinda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hakkında</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>dht11_700p.png</normaloff>dht11_700p.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>-330</x>
      <y>530</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>591</width>
      <height>171</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;BİLAL YAŞAR 1612710001 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;1.ÖĞRETİM &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; GÖRSEL PROGRAMLAMA DERSİ&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt; DÖNEM ÖDEVİ&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;br /&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>205</y>
      <width>591</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Giriş Sayfası İçin Tıklayın</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionGiri"/>
    <addaction name="actionKay_t_Ol"/>
    <addaction name="action_k"/>
   </widget>
   <addaction name="menuProgram"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGiri">
   <property name="text">
    <string>Giriş</string>
   </property>
  </action>
  <action name="actionKay_t_Ol">
   <property name="text">
    <string>Kayıt Ol</string>
   </property>
  </action>
  <action name="action_k">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
